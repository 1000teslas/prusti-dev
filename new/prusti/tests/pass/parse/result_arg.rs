#![feature(register_tool)]
#![register_tool(prusti)]

#[prusti::requires(true)]
pub fn test1(result: u32) -> u32 { result }

fn main() {}
