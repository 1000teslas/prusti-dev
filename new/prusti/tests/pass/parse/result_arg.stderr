error: "it is not allowed to use the keyword `result` as a function argument"
 --> $DIR/result_arg.rs:5:14
  |
5 | pub fn test1(result: u32) -> u32 { result }
  |              ^^^^^^^^^^^

error: aborting due to previous error

